app/views/results/show.html.erb

```diff
@@ -6,52 +6,56 @@
 
 <div class="container-fluid">
   <div id="result" data-result-id="<%=@result.id%>">
-      <div class="row">
-        <div class="col-12">
-          <h1 class="my-4 text-center">Game Results</h1>
+    <div class="row">
+      <div class="col-12">
+        <h1 class="my-4 text-center">Game Results</h1>
 
-          <!-- Partial for Imposter reveal -->
-          <div id="imposter-reveal" class="d-flex justify-content-center align-items-center result-page my-2 text-center" style="height:10vh;">
-            <h4 class="px-2">Waiting for other players: <%= @total_votes %> / <%= @total_voters %></h4>
-          </div>
-
-          <!-- Partial for chart -->
-          <div id="chart" class="result-page my-2" style="height:40vh; overflow: auto;">
-            <%= render  "result_chart", players: @players %>
-          </div>
+        <!-- Partial for Imposter reveal -->
+        <div id="imposter-reveal" class="d-flex justify-content-center align-items-center result-page my-2 text-center" style="height:10vh;">
+          <h4 class="px-2">Waiting for other players: <%= @total_votes %> / <%= @total_voters %></h4>
+        </div>
 
+        <!-- Partial for chart -->
+        <div id="chart" class="result-page my-2" style="height:40vh; overflow: auto;">
+          <%= render  "result_chart", players: @players %>
         </div>
-      </div>
-      <div class="row">
-        <div class="col-12">
 
         <!-- Partial for Vote Tally -->
-          <div id="vote-tally" class="result-page" style="height:20vh;">
-            <%= render  "result_vote_tally" %>
-          </div>
-
+        <div id="vote-tally" class="result-page" style="height:20vh;">
+          <%= render  "result_vote_tally" %>
         </div>
       </div>
     </div>
+  </div>
 
-      <div class="row">
-        <div class="col-12">
-          <div id="result-footer" class="d-flex justify-content-around my-3" style="height:5vh;" data-html2canvas-ignore>
-            <% if @current_user.id == @instance.user_id %>
-              <%= simple_form_for Instance.new do |f| %>
-                <%# needed for creating a new instance %>
-                <div>
-                  <%# needed to get last_instance_id for creation of new instance %>
-                  <%= hidden_field_tag 'last_instance_id', @instance.id %>
-                  <%= f.submit "Exit Game", class: "btn btn-primary" %>
-                </div>
-              <% end %>
-            <% end %>
 
-            <%= render "share_modal" %>
-          </div>
+  <!-- This shouldn't get refreshed -->
```

Comment:
New div for SNS buttons and continue button so that they are not affected by AJAX refreshing